src/server/test/hotspot_partition_test.cpp

```diff
@@ -126,6 +132,9 @@ TEST_F(hotspot_partition_test, hotspot_partition_policy)
 
 TEST_F(hotspot_partition_test, send_detect_hotkey_request)
 {
+    auto default_occurrence_threshold = FLAGS_occurrence_threshold;
+    FLAGS_occurrence_threshold = 100;
```

Comment:
Why do you update `occurrence_threshold` default value to 3 in `hotspot_partition_calculator.cpp`, and update it 100 in unit test?